chia/consensus/blockchain.py

```diff
@@ -628,14 +629,15 @@ async def validate_unfinished_block(
         )
 
         if error_code is not None:
-            return PreValidationResult(uint16(error_code.value), None, None)
+            return PreValidationResult(uint16(error_code.value), None, None, False)
 
-        return PreValidationResult(None, required_iters, cost_result)
+        return PreValidationResult(None, required_iters, cost_result, False)
 
     async def pre_validate_blocks_multiprocessing(
         self,
         blocks: List[FullBlock],
         npc_results: Dict[uint32, NPCResult],
+        validate_signatures: bool,
```

Comment:
I think it would be safer to add `,*` before this, to force all callers to name the parameter. If there are calls to this function where the `batch_size` is passed in as positional, are you sure it couldn't be interpreted as a boolean?

Reply:
Good idea